src/app/components/ExternalLinksCard.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import {
+  Title,
+  Bullseye,
+  Split,
+  Flex,
+  FlexItem,
+  Card,
+  CardHeader,
+  CardBody,
+  CardFooter,
+  Button,
+  Grid,
+  GridItem,
+  CardTitle
+} from '@patternfly/react-core'
+
+
+import redhat_logo from "@app/bgimages/redhat_clear.png";
+import aws_logo from "@app/bgimages/aws_clear.png";
+import azure_logo from "@app/bgimages/azure_clear.png";
+import google_logo from "@app/bgimages/google_clear.png";
+
+
+const providerInfos = [
+  {
+    name: 'aws',
+    text: 'Launch in AWS ->',
+    url: 'https://aws.amazon.com/marketplace/pp/prodview-kv5mi3ksb2mma?sr=0-1&ref_=beagle&applicationId=AWSMPContessa'
+  },
+  {
+    name: 'google',
+    text: 'Launch in Google Cloud ->',
+    url: 'https://console.cloud.google.com/marketplace/product/rhel-cloud/rhel-9?project=cockpituous'
+  },
+  {
+    name: 'azure',
+    text: 'Launch in Azure ->',
+    url: 'https://azuremarketplace.microsoft.com/en-us/marketplace/apps/redhat.rhel-20190605?tab=Overview'
+  }
+]
+
+const loadImage = (name) => {
+  switch (name) {
+    case 'aws':
+      return aws_logo
+    case 'azure':
+      return azure_logo
+    case 'google':
+      return google_logo
+  }
+}
+
+const ExternalLinksCard: React.FunctionComponent = () => {
+  return (
+    <Flex direction={{default: 'column'}} flex={{default: 'flex_3'}}
+          justifyContent={{default: 'justifyContentSpaceBetween'}}
+          style={{border: '1px solid #D2D2D2', padding: '2%'}}>
```

Comment:
So it seems to me that this styling is really just re-implementing a card. What if you used a card from Patternfly here instead?

https://www.patternfly.org/v4/components/card/So it seems to me that this styling is really just re-implementing a card. What if you used a card from Patternfly here instead?

https://www.patternfly.org/v4/components/card/

Now, the card does have a shadow and your design does not. There is a 95% chance that this is fine. This is where if you tell your designer "Hey, you know the PF card has a shadow by default!" there is a good chance they will say "Oh, okay, let's go with that!". 

If they say "No, no shadow here for {REASONS}", then you can use CSS (for the card!) to remove the border. The PF documentation has the specific CSS selectors:
https://www.patternfly.org/v4/components/card/#css-variables

Reply:
Actually, regarding the shadow, all you need to do is pass in the `isFlat` prop and it will look exactly like the designs! https://www.patternfly.org/v4/components/card#modifiers